NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -94,6 +95,9 @@ public enum NeuroID {
         
         // Reset tab id on configure
         setUserDefaultKey(Constants.storageTabIDKey.rawValue, value: nil)
+
+
+        locationManager = LocationManager()
```

Comment:
Do we need to a getter for the locationManager so the client can authorize its use? ```  locationManager.requestWhenInUseAuthorization()```

Reply:
I wasn't asked permission when I tried this on my device and I only see the defaults of -1.0 for lat long in the logs. So it prob needs permission.

Reply:
Yeah I was seeing this too. It looks like you have to call requestWhenInUseAuthoriztion() else the self.Authoriztion status is not determined and the location manager is not started.

Reply:
Correct, but we aren't concerned with asking for permission, we just want to see if permission is granted by the user and then collect. We assume our customer will be asking for the location permissions and we just want to collect based on allowed scopes. I can show you both a demo that gets location if you'd like